app/serializers/organization_serializer.rb

```diff
@@ -1,26 +1,37 @@
 class OrganizationSerializer
   include RestPack::Serializer
   include ContentSerializer
+  include MediaLinksSerializer
 
-  attributes :id, :name, :display_name, :description, :introduction, :title, :href
+  attributes :id, :name, :display_name, :description, :introduction, :title, :href, :avatar_src
```

Comment:
It would be nice to have the avatar_src as an optional attribute e.g. [here](https://github.com/zooniverse/Panoptes/blob/bcfc0b2d659a533e71ee7d3f7774379e78e4d36d/app/serializers/project_serializer.rb#L32) instead of on each request. Mostly as this is duplicate info via the `media_include :avatar` call below